op-challenger/game/fault/contracts/oracle.go

```diff
@@ -24,6 +25,14 @@ const (
 	methodProposalCount             = "proposalCount"
 	methodProposals                 = "proposals"
 	methodProposalMetadata          = "proposalMetadata"
+	methodProposalBlocksLen         = "proposalBlocksLen"
+	methodProposalBlocks            = "proposalBlocks"
+
+	LeafSize = 136
```

Comment:
In a previous PR, I introduced a `LeafSize` variable in this package as well but you had said it doesn't make sense since we already have one defined in the `keccak/matrix` package. Should we just be using `matrix.LeafSize` here instead?

Reply:
Another possibility is to hoist `LeafSize` up one level to the `game` package types as a `KeccakLeafSize` type so there's less potential dependency conflicts between the `keccak` and `fault` packages. I'm not strongly opinionated here since I don't have intuition on what type of games will be supported in the future here if at all.

Reply:
Ah I see this is done in your follow up pr #9123, looks much better so this can be dismissed